index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portifolio</title>
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <header class="toolbar">
+    <a href="index.html" class="name button"> SOLOMON</a>
+    <nav>
+      <img  class="huberger" onclick="openNav()" src="/assets/Union.png" alt="Dropdown menu">
```

Comment:
- [ ] Your hamburger button isn't visible. Kindly try to update your relative source path to fix it.